components/count-badge/README.md

```diff
@@ -62,28 +53,30 @@ The `d2l-count-badge-icon` element is a web component to display a number count,
-| `hide-zero`, default: `false` | Boolean | Optionally choose not to show the count badge when the number is zero. |
-| `text`  | String | REQUIRED: Descriptive text for the badge which will act as an accessible label and tooltip text when tooltips are enabled. |
-| `tab-stop`, default: `false` | Boolean | Optionally choose to make the badge a tab stop. |
-| `announce-changes`, default: `false` | Boolean | Optionally choose to announce changes to the badge with an aria-live region. If the text property is changed, the text will be read by screen-readers. |
-| `has-tooltip`, default: `false` | Boolean | Optionally choose to have a tooltip below the badge. |
-| `focus-ring`, default: `false` | Boolean | Optionally force a focus ring around the badge. This property can be used to highlight the badge when the parent element is focused. |
+| `number` | Number, required | The number to display on the badge, must be a positive integer |
+| `icon` | String, required | [Preset icon key](../icons#preset-icons) (e.g. `tier1:gear`) |
+| `size` | String, default: `small` | The size of the badge. Valid options are `"small"` and `"large"`. |
+| `type` | String, default: `count` | The type of the badge. Valid options are `"notification"` and `"count"`. Notification badges are orange and count badges are grey. |
+| `max-digits` | Number, default: `2` when `type="notification"`, `5` when `type="count"` | Specifies a digit limit, after which numbers are truncated. Defaults to two for `"notification"` type and five for `"count"` type. Must be between 1-5.
+| `hide-zero` | Boolean, default: `false` | Hides the count badge when `number` is zero |
+| `text` | String, required | Descriptive text for the badge which will act as an accessible label and tooltip text when tooltips are enabled |
+| `tab-stop` | Boolean, default: `false` |  Adds a tab stop to the badge, which allows screen reader and keyboard users to easily tab to the badge |
+| `announce-changes` | Boolean, default: `false` | When `true`, changes to the badge will be announced to screen reader users |
+| `has-tooltip` | Boolean, default: `false` | Adds a tooltip on the badge, which will be visible on hover/tab-stop, and read out by screen-readers |
+| `focus-ring` | Boolean, default: `false` | Forces a focus ring around the badge. This property can be used to highlight the badge when the parent element is focused. |
 <!-- docs: end hidden content -->
+
+## Accessibility
+
+Notable accessibility features:
+- `tab-stop`:
+  - While non-interactable components traditionally shouldn't be focusable, this property greatly helps screen reader users find the the count badge, and removes the need to use the arrow keys to navigate to it.
+  - If the optional tooltip is used, it isn't necessary, as the count-badge becomes focusable in order to display the tooltip
```

Comment:
```suggestion
  - This isn't needed if the optional tooltip is used, as the count-badge becomes focusable in order to display the tooltip.
```

Reply:
I've made all these changes, with the exception of having the period at the end of the line here, since I remember you mentioning that we shouldn't be including them. Unless there's a special case when it's a longer sentence?

Reply:
I added it here to be consistent with the line above. I do wonder about the longer sentence piece. Thoughts @geurts ?

Reply:
Based on https://daylight.d2l.dev/style-elements/writing/#periods it should probably just be left off, so LGTM